content/en/platform/corda/5.0/deploying-operating/deployment/deploying/manual-bootstrapping.md

```diff
@@ -250,9 +272,62 @@ To create the schema manually, do the following:
    {{% /tab %}}
    {{< /tabs >}}
 
-5. Review the DDL files generated and then execute against the database.
+2. Review the DDL files generated and then execute against the database, ensuring to apply them to the `CONFIG` schema.
 
-6. Execute the following Corda CLI command to generate DDL for populating the Crypto database connection configuration, in this case use CONFIG:
+#### RBAC Database Connection Configuration for Corda Enterprise {{< enterprise-icon >}}
+
+1. Execute the following Corda CLI command to generate DDL for populating the RBAC database connection configuration:
+
+   {{< tabs name="RBAC-ent">}}
+   {{% tab name="Bash" %}}
+   ```sh
+   corda-cli.sh initial-config create-db-config -u <RBAC-USERNAME> \
+      --name corda-rbac --jdbc-url 'jdbc:postgresql://<DB-HOST>:<DB-PORT>/<DB=NAME>?currentSchema=RBAC' \
+      --jdbc-pool-max-size <POOL-SIZE> -t VAULT --vault-path dbsecrets --key rbac -l /tmp/db
```

Comment:
Again, I think `dbsecrets` in these commands just needs to match whatever they give in their configuration overrides so we should probably show this as a variable. You already call out that this configuration needs to match what is in the overrides.